### PR TITLE
[DESK][SHELLBTRFS] Fix typo in WIN32_NO_STATUS

### PR DESCRIPTION
**Pull request by HBelusca**
"WIN32_NO_STAT _**u**_ S" -> "WIN32_NO_STATUS"

### CODE REVIEW

**Review by binarymaster — APPROVED** [review]

File: dll/shellext/shellbtrfs/guid.c [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* DO NOT USE THE PRECOMPILED HEADER FOR THIS FILE! */
 
-#define WIN32_NO_STATuS
+#define WIN32_NO_STATUS
```
Comment (binarymaster) [inline dll/shellext/shellbtrfs/guid.c:3]: Is upstream file affected by this?

Comment (SergeGautherie) [reply dll/shellext/shellbtrfs/guid.c:3]: > upstream file

This is a ReactOS file.

**Comment by SergeGautherie** [comment]
As it works without the proper define, I assume this is mostly a compilation optimization, isn't it?

While here, you may want to check/fix ['ntstatus.s from NDK' comments](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ntstatus.s):
Should they be 'ntstatus.h from PSDK'?

**Review by GeoB99 — APPROVED** [review]

**Comment by HBelusca** [comment]
> As it works without the proper define, I assume this is mostly a compilation optimization, isn't it?

Probably, yes.

> While here, you may want to check/fix ['ntstatus.s from NDK' comments](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=ntstatus.s): Should they be 'ntstatus.h from PSDK'?

See PR #5771

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**